.github/workflows/deployment.yml

```diff
@@ -75,10 +83,14 @@ jobs:
     - name: Package application image
       run: ${{ matrix.archivePortable }}
       shell: bash
-    - name: Extract branch name
+    - name: Build and publish snap
+      if: matrix.os == 'ubuntu-latest' && steps.extract_branch.outputs.branch == 'master'
+      env:
+        SNAPCRAFT_LOGIN_FILE: ${{ secrets.SNAPCRAFT_LOGIN_FILE }}
+      run: |
+          docker run -v $(pwd):$(pwd) -t lyzardking/snapcraft-bionic sh -c "apt update -qq && cd $(pwd) && snapcraft && mv jabref*.snap build/distribution/"
+          # cd build/distribution/ && mdkir .snapcraft && echo ${SNAPCRAFT_LOGIN_FILE} | base64 --decode --ignore-garbage > .snapcraft/snapcraft.cfg && snapcraft push --release=beta *.snap
```

Comment:
This part needs secrets and activations as proposed in https://github.com/JabRef/jabref/pull/5379#issuecomment-537887016

Reply:
@koppor can you have a look here? thanks!